app/controllers/organizations/adopter_fosterer/adopter_applications_controller.rb

```diff
@@ -26,7 +26,7 @@ def create
         .new_adoption_application.deliver_now
     else
       redirect_to adoptable_pet_path(@pet),
-        alert: "Error. Please try again."
+        alert: t(".error")
```

Comment:
So this finds the appropriately scoped translation? I have always called the full reference to it and did not know this would work!

Reply:
Yes, it's called lazy lookup. Important to notice that it is not ALWAYS suitable. Sometimes we have to use the full path. In tests or module helpers requires `I18n.t("full/path"). 

ATE I18n rails documentation for this moment🤣